test/project_types/extension/features/argo_runtime_test.rb

```diff
@@ -7,105 +7,28 @@ module Features
-          assert_equal accepts_port, runtime.supports?(:port)
-        end
-      end
-
-      def test_supports_public_url
-        runtimes = {
-          checkout_runtime => supports_feature,
-          admin_runtime => supports_feature,
-        }
-
-        runtimes.each do |runtime, accepts_tunnel|
-          assert_equal accepts_tunnel, runtime.supports?(:public_url)
-        end
-      end
-
-      def test_supports_uuid
-        runtimes = {
-          checkout_runtime => does_not_support_feature,
-          admin_runtime => supports_feature,
-        }
-
-        runtimes.each do |runtime, accepts_uuid|
-          assert_equal accepts_uuid, runtime.supports?(:uuid)
-        end
-      end
-
-      def test_supports_renderer_version
-        runtimes = {
-          checkout_runtime => does_not_support_feature,
-          admin_runtime => supports_feature,
-        }
-
-        runtimes.each do |runtime, accepts_renderer_version|
-          assert_equal accepts_renderer_version, runtime.supports?(:renderer_version)
-        end
-      end
-
-      def test_supports_api_key
-        runtimes = {
-          checkout_runtime => does_not_support_feature,
-          admin_runtime => supports_feature,
-        }
-
-        runtimes.each do |runtime, accepts_api_key|
-          assert_equal accepts_api_key, runtime.supports?(:api_key)
-        end
-      end
-
-      def test_supports_shop
-        runtimes = {
-          checkout_runtime => does_not_support_feature,
-          admin_runtime => supports_feature,
-        }
-
-        runtimes.each do |runtime, accepts_shop|
-          assert_equal accepts_shop, runtime.supports?(:shop)
-        end
-      end
-
-      def test_supports_name
-        runtimes = {
-          checkout_runtime => does_not_support_feature,
-          admin_runtime => supports_feature,
-        }
-
-        runtimes.each do |runtime, accepts_name|
-          assert_equal accepts_name, runtime.supports?(:name)
-        end
-      end
-
-      private
-
-      def checkout_runtime
-        ArgoRuntime.new(
-          cli: Models::NpmPackage.new(name: "@shopify/checkout-ui-extensions-run", version: "0.4.0"),
-          renderer: Models::NpmPackage.new(name: "@shopify/post-purchase-ui-extensions", version: "0.9.0")
+      def test_admin_runtime_is_returned_for_admin_extensions
+        runtime = ArgoRuntime.build(
+          cli_package: Models::NpmPackage.new(name: "@shopify/admin-ui-extensions-run", version: "0.13.0"),
+          identifier: "PRODUCT_SUBSCRIPTION"
         )
+        assert_equal(Runtimes::Admin, runtime.class)
       end
 
-      def admin_runtime
-        ArgoRuntime.new(
-          cli: Models::NpmPackage.new(name: "@shopify/admin-ui-extensions-run", version: "0.13.0"),
-          renderer: Models::NpmPackage.new(name: "@shopify/admin-ui-extensions", version: "0.12.0")
+      def test_checkout_ui_extension_runtime_is_returned_for_checkout_ui_extensions
```

Comment:
Should we add a similar test with the "legacy" identifier, `CHECKOUT_ARGO_EXTENSION`?